utils.ts

```diff
@@ -9,6 +9,28 @@ import shelljs from "shelljs";
 import { createServer } from "vite";
 import chalk from "chalk";
 import { generate } from "./generate";
+import axios, { Method } from "axios";
+import { Stream } from "stream";
+
+export async function makeCloudApiRequest(cloudApiBase: string, path: string, token: string, method: Method = "GET") {
+  try {
+    const response = await axios({
+      method,
+      baseURL: cloudApiBase,
+      url: path,
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      responseType: "stream",
+    });
+
+    response.data.pipe(process.stdout);
+  } catch (err) {
+    if (axios.isAxiosError(err)) {
+      (err.response?.data as Stream).on("data", (data) => console.error(data.toString()));
+    }
+  }
```

Comment:
should probably log the err if it's not an axois error (also in deploy)

Reply:
done